vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/video/ScreenRecorder.kt

```diff
@@ -32,7 +31,65 @@ internal class ScreenRecorder(
         logger.debug { "Recording finished in ${millis}ms $remoteFilePath" }
     }
 
+    suspend fun stopScreenRecord() {
+        logger.debug { "Stopping screen recorder" }
+        var hasKilledScreenRecord = true
+        var tries = 0
+        while (hasKilledScreenRecord && tries++ < SCREEN_RECORD_KILL_ATTEMPTS) {
```

Comment:
I recommend keeping the rule "one line - one action."
Could you put `tries++` in a separate line?

Reply:
Not a `++` fan I assume